pkg/plugins/gateway/cloudrun/http.go

```diff
@@ -47,6 +49,9 @@ func middleware(ctx *fasthttp.RequestCtx, pool worker.WorkerPool) bool {
 	if err := json.Unmarshal(bodyBytes, &pubsubEvent); err == nil && pubsubEvent.Subscription != "" {
 		// We have an event from pubsub here...
 
+		topic := pubsubEvent.Message.Attributes["x-nitric-topic"]
+		ctx := span.FromHeaders(context.TODO(), "topic-"+topic, triggers.HttpHeaders(&rc.Request.Header))
```

Comment:
Does this work? I can't see where otel propagation would be occurring using HTTP headers for pubsub message publishing. What tells pubsub to apply these HTTP headers to the subscription?